ccm_web/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Establish base image to build upon
+FROM node:14-slim AS node
+WORKDIR /app/
+
+EXPOSE 3000
```

Comment:
I would suggest to keep the port different from 3000 since it is interfering with Myla ports

Reply:
Change this to 3333 and do
```
    ports:
      - 3333:3333
```
in docker-compose.yml and the startup link works as well as not being on 3000

Reply:
Sure, I can change to `3333` if you both think that makes sense.

Reply:
I don't know that making a decision here based on myla is particularly important, but its just a number so what does it hurt.  
I  mostly want to be able to click the link produced on startup and matching the ports up in docker-compose.yaml seems to fix that part of it.

Reply:
K, I don't know what the best practice is here. I'm not aware of any particular issues with mapping to the same port, so sure, I'll do it.

Reply:
Done. See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/8/commits/7bd104a36edce63c8fcffab4b4e41bb18767f34a